index.html

```diff
@@ -124,30 +125,27 @@ <h3>Featured Speakers</h3>
         <div class="container experts">
         </div>
       </section>
-      <section id="footer">
+    <section id="footer">
         <div class="advert">
           <h6>Partners</h6>
           <hr>
           <div class="advlogo">
             <h2>Mozilla</h2>
             <h3>Google</h3>
             <h2>NAVER</h2>
-            <p>daumkakao</p><img src="./images/Icons/clip.png" alt="clip">airbnb</p>
+            <p>daumkakao</p><p><img src="./images/Icons/clip.png" alt="clip">airbnb</p>
           </div>
         </div>
-        <div class="copyright"><img src="./images/Logo.png">EPALTechs<a alt="Epal Logo" style="width:60px;"></a>
+        <div class="copyright"><a></a><img src="./images/Logo.png" alt="Epal Logo"> EPALTechs
           <div class="details">
-            <h5>2022 Conference on Web Development.</h5>
+            <h5>2022 Conference on Web Development.<span>All Rights Reserved.</span></h5>
             <p>
-              Epal—Technologies unusual. Concepts and ideas curated by the stakeholders of the company, domicile in Uyo
+              Epal—Technologies unusual. Concepts and ideas shared by the stakeholders of the company, domicile in Uyo
               City, Akwa Ibom State, Nigeria.
             </p>
           </div>
         </div>
-        </div>
-      </section>
+    </section>
```

Comment:
Hi @Epaltechs, 
I know that the images don't show these sections but what you have in Behance is a slice of the page, not the whole page and it's just not right to have a page without a footer and a page with a footer 😉 so please show them in your mobile view for the home page.
and Kindly implement all the required changes because this is your first review for this project this week.

Have a great day.